src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
```

Comment:
This file can probably be reverted (only change is adding some `using`s).

Reply:
It's intentional.  InvertIf moved to an appropriate location.  It was strangely placed underneath a subfolder instead of having a top level feature directory (like all other 'Feature' features).  I noticed that while doing this work, so i fixed alongside this (it's the first commit).